tests/performance/playwright.config.js

```diff
@@ -19,9 +19,7 @@ process.env.TEST_RUNS ??= '20';
 const config = defineConfig( {
 	...baseConfig,
 	globalSetup: require.resolve( './config/global-setup.js' ),
-	reporter: process.env.CI
-		? './config/performance-reporter.js'
-		: [ [ 'list' ], [ './config/performance-reporter.js' ] ],
+	reporter: [ [ 'list' ], [ './config/performance-reporter.js' ] ],
```

Comment:
What's the rationale for this change?

Reply:
It's an oversight from the initial implementation. Without this, any errors normally thrown by Playwright are simply swallowed. There is no good reason to do this.